packs/st2/actions/lib/action.py

```diff
@@ -47,7 +47,7 @@ def _get_auth_token(self):
 
         return token
 
-    def _run_client_method(self, method, method_kwargs, format_func):
+    def _run_client_method(self, method, method_kwargs, format_func, format_kwargs={}):
```

Comment:
Should default to a non mutable type (e.g. None and do format_kwargs = format_kwargs or {} in the body)/